include/aspect/simulator.h

```diff
@@ -597,6 +597,7 @@ namespace aspect
       void
       local_assemble_advection_system (const AdvectionField &advection_field,
                                        const std::pair<double,double> global_field_range,
+                                       Vector<double> &viscosity_per_cell,
```

Comment:
Please make this variable const and indent the variable name to the same column as the other ones.